site/docs/nodejs/installion.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 2
+---
+
+# Installing Node.js and Discordeno
+To use the Discordeno library you would need to install Node.js and then Discordeno from NPM.
+
+Go on this [website](https://nodejs.org/en/) and download the latest version of Node.js.
+Then open the downloaded file and follow the instructions of the installer to install Node.js.
+
+## Create a Folder
+Open your File Explorer and create a new folder (Ex: `discordbot`) in your wished directory.
+After this open your Coding IDE and create a new file (Ex: `index.js`) in the folder you just created.
+
+### Initalize NPM
+Inorder to keep track of the dependencies, you need to initialize NPM, which creates a `package.json` file.
+```cli
+npm init
```

Comment:
@Skillz4Killz it's `init` (also maybe use the full word "install" and "--yes" so that it's easier to understand?)
`npm init --yes ; npm install discordeno`